packages/langium-cli/src/generator/module-generator.ts

```diff
@@ -41,12 +42,13 @@ export function generateModule(grammars: Grammar[], config: LangiumConfig, gramm
             grammarsWithName,
             grammar => {
                 const config = grammarConfigMap.get(grammar)!;
+                const modeValue = mode ? `,${EOL}    mode: '${mode}'` : '';
                 return expandToNode`
 
                     export const ${ grammar.name }LanguageMetaData = {
                         languageId: '${config.id}',
                         fileExtensions: [${config.fileExtensions && joinToNode(config.fileExtensions, e => appendQuotesAndDot(e), { separator: ', ' })}],
-                        caseInsensitive: ${Boolean(config.caseInsensitive)}
+                        caseInsensitive: ${Boolean(config.caseInsensitive)}${modeValue}
```

Comment:
This and the definition of `modeValue` are a bit unreadable. How about something like this:
```suggestion
                        caseInsensitive: ${Boolean(config.caseInsensitive)},
                        mode: '${mode ?? 'development'}',
```
or factor the `?? 'development'` part out to `modeValue`